db/models.py

```diff
@@ -50,3 +55,54 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey("User", on_delete=models.CASCADE,
+                             related_name="orders")
+
+    class Meta:
+        ordering = ("-created_at",)
+
+    def __str__(self) -> str:
+        return f"{self.created_at}"
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(MovieSession, on_delete=models.CASCADE,
+                                      related_name="tickets")
+    order = models.ForeignKey(Order, on_delete=models.CASCADE,
+                              related_name="tickets")
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    class Meta:
+        constraints = [models.UniqueConstraint(fields=["movie_session",
+                                                       "row", "seat"],
+                                               name="unique_session-row-seat")]
+
+    def __str__(self) -> str:
+        return f"{self.movie_session} (row: {self.row}, seat: {self.seat})"
+
+    def clean(self) -> None:
+        if not (1 <= self.row <= self.movie_session.cinema_hall.rows):
+            raise ValidationError({
+                "row": "row number must be in available range: "
+                       "(1, rows): "
+                f"(1, {self.movie_session.cinema_hall.rows})"
+            })
+        if not (1 <= self.seat <= self.movie_session.cinema_hall.seats_in_row):
+            raise ValidationError({
+                "seat": "seat number must be in available range: "
+                        "(1, seats_in_row): "
+                f"(1, {self.movie_session.cinema_hall.seats_in_row})"
+            })
+
+    def save(self, *args, **kwargs) -> None:
+        self.full_clean()
+        return super().save(*args, **kwargs)
+
+
+class User(AbstractUser):
+    pass
```

Comment:
The `User` model extends `AbstractUser` but does not add any additional fields or methods. If you intend to customize the user model further, consider adding any additional fields or methods here. Otherwise, ensure that this model is correctly set as the custom user model in your Django settings.